couchcopy

```diff
@@ -460,11 +488,6 @@ async def main():
     sub['backup'].add_argument('couchdb',
                                metavar='hostname,/couchdb/data/dir')
     sub['backup'].add_argument('archive', metavar='backup.tar.gz')
-    sub['backup'].add_argument('--rsync-reuse-dir', help=(
```

Comment:
Did you consider keeping both the old `rsync` way and the new `tar over ssh` way at the same time? Maybe there's somebody out there relying on these options :stuck_out_tongue: